test/e2e/fixture/scylla/scylladbmonitoring.yaml.tmpl

```diff
@@ -0,0 +1,35 @@
+apiVersion: scylla.scylladb.com/v1alpha1
+kind: ScyllaDBMonitoring
+metadata:
+  name: "{{ .name }}"
+spec:
+  endpointsSelector:
+    matchLabels:
+      app.kubernetes.io/name: scylla
+      scylla-operator.scylladb.com/scylla-service-type: identity
+      scylla/cluster: "{{ .scyllaClusterName }}"
+  components:
+    prometheus:
+      exposeOptions:
+        webInterface:
+          ingress:
+            ingressClassName: haproxy
```

Comment:
shlouldn't ingressClassName and annotations be parametrized via CLI parameters? We don't rely on any particular ingress implementation, it can be provided by user or CI depending what's deployed.

Reply:
you are correct - do you want to build on top of this in your PR (#1076) so I don't have to duplicate it here?

Reply:
all three parameters are already present in #1076, do a review there and we can merge and rebase here.

Reply:
Although you are logically correct, I was proposing a different order because of priorities though. Would that be fine with you?

Reply:
please add a comment in #1076 to not forget about it once this one is merged.

Reply:
https://github.com/scylladb/scylla-operator/pull/1076#issuecomment-1400578701